Sources/CryptoSwift/AES.swift

```diff
@@ -307,19 +307,14 @@ fileprivate extension AES {
         var rk2: Array<Array<UInt32>> = expandKey(key, variant: variant)
 
         for r in 1 ..< rounds {
-            var w: UInt32
-
-            w = rk2[r][0]
-            rk2[r][0] = U1[Int(B0(w))] ^ U2[Int(B1(w))] ^ U3[Int(B2(w))] ^ U4[Int(B3(w))]
-
-            w = rk2[r][1]
-            rk2[r][1] = U1[Int(B0(w))] ^ U2[Int(B1(w))] ^ U3[Int(B2(w))] ^ U4[Int(B3(w))]
-
-            w = rk2[r][2]
-            rk2[r][2] = U1[Int(B0(w))] ^ U2[Int(B1(w))] ^ U3[Int(B2(w))] ^ U4[Int(B3(w))]
-
-            w = rk2[r][3]
-            rk2[r][3] = U1[Int(B0(w))] ^ U2[Int(B1(w))] ^ U3[Int(B2(w))] ^ U4[Int(B3(w))]
+            for i in 0..<4 {
+                let w = rk2[r][i]
+                let u1 = U1[Int(B0(w))]
+                let u2 = U2[Int(B1(w))]
+                let u3 = U3[Int(B2(w))]
+                let u4 = U4[Int(B3(w))]
+                rk2[r][i] = u1^u2^u3^u4
```

Comment:
Other than running the unit tests I didn't run any performance tests. I think we gained something by removing that `var` but I do see now how that `0..<4` loop could affect performance if the compiler isn't smart enough to optimize a "constant" range. I can break those out into 4 copies.